nav2_smac_planner/src/smac_planner_hybrid.cpp

```diff
@@ -221,6 +222,7 @@ void SmacPlannerHybrid::configure(
   _raw_plan_publisher = node->create_publisher<nav_msgs::msg::Path>("unsmoothed_plan", 1);
   if (_viz_expansions) {
```

Comment:
Change this to `_debug_visualizations` and then change the parameter name to `debug_visualizations` as well if its going to do more than just visualize expansions. I think that's a good way to go, we just need to change the name to be proper

Reply:
Thank you for your patient guidance，i will modify it as soon as possible.